docs/scripts/buildApi.ts

```diff
@@ -0,0 +1,261 @@
+import * as yargs from 'yargs';
+import * as TypeDoc from 'typedoc';
+import { writeFileSync, mkdirSync } from 'fs';
+import path from 'path';
+import kebabCase from 'lodash/kebabCase';
+import * as prettier from 'prettier';
+import { renderInline as renderMarkdownInline } from '../../node_modules/@material-ui/monorepo/docs/src/modules/utils/parseMarkdown';
```

Comment:
This could work 

```suggestion
import { renderInline as renderMarkdownInline } from '@material-ui/monorepo/docs/src/modules/utils/parseMarkdown';
```